samples/msgext-ai-doc-compliance-checker/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+    {
+      "name": "officedev-microsoft-teams-samples-msgext-ai-doc-compliance-checker-js",
```

Comment:
This name should be properly added in readme Fragment section on top table.

Reply:
Added